### PR TITLE
filter field for google windows

### PR DESCRIPTION
**Pull request by pavertomato**
filter field for google windows

### CODE REVIEW

File: platform/viewer/src/googleCloud/LocationsList.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ class LocationsList extends Component {
 
     const body = (
       <tbody id="LocationList">
-        {this.props.locations.map(this.renderTableRow)}
+        {this.props.locations.filter(location => (location.name.split('/')[3].toLowerCase().includes(this.props.filter.toLowerCase()) || this.props.filter=="")).map(this.renderTableRow)}
```
Comment (dannyrb) [inline platform/viewer/src/googleCloud/LocationsList.js]: Can we destructure the props higher up so this one-liner is easier to read?

File: platform/viewer/src/googleCloud/LocationPicker.js [hunk 2]
```diff
@@ -37,15 +38,22 @@ export default class LocationPicker extends Component {
...
-        locations={locations}
-        loading={loading}
-        error={error}
-        onSelect={onSelect}
-      />
+      <div>
+        <input class="form-control gcp-input"
+            type="text"
+            value={ filterStr }
+            onChange={ e => this.setState({ filterStr: e.target.value }) } />
```
Comment (dannyrb) [inline platform/viewer/src/googleCloud/LocationPicker.js]: Can we move the closing portion of this tag to the next line? `/>`

File: platform/viewer/src/googleCloud/DicomStorePicker.js [hunk 3]
```diff
@@ -38,16 +39,23 @@ export default class DicomStorePicker extends Component {
...
-        stores={stores}
-        loading={loading}
-        error={error}
-        onSelect={onSelect}
-      />
+      <div>
+        <input class="form-control gcp-input"
+            type="text"
+            value={ filterStr }
+            onChange={ e => this.setState({ filterStr: e.target.value }) } />
```
Comment (dannyrb) [inline platform/viewer/src/googleCloud/DicomStorePicker.js]: Similarly, can we move the closing portion of this tag to the next line? `/>`

**Comment by dannyrb** [comment]
@pavertomato can you include a screenshot of what this change introduces? I think the gist is the ability to filter projects, datasets, etc.

If we destructure props, and clean up the formatting then this LGTM 👍

File: platform/viewer/src/googleCloud/LocationsList.js (hunk 1 above)
Comment (pavertomato) [reply platform/viewer/src/googleCloud/LocationsList.js]: done

File: platform/viewer/src/googleCloud/DicomStorePicker.js (hunk 3 above)
Comment (pavertomato) [reply platform/viewer/src/googleCloud/DicomStorePicker.js]: done

**Comment by pavertomato** [comment]
> @pavertomato can you include a screenshot of what this change introduces? I think the gist is the ability to filter projects, datasets, etc.
> 
> If we destructure props, and clean up the formatting then this LGTM +1


65ce9b83a992

**Pull request merged**